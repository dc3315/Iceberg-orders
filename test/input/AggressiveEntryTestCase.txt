B,100324,99,50000
S,100323,101,20000
S,100325,100,10000
S,100326,100,7500
B,100327,98,25500
B,100327,100,100000,10000