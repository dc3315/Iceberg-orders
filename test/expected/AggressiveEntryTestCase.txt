+-----------------------------------------------------------------+
| BUY                            | SELL                           |
| Id       | Volume      | Price | Price | Volume      | Id       |
+----------+-------------+-------+-------+-------------+----------+
|    100324|       50,000|     99|       |             |          |
+----------+-------------+-------+-------+-------------+----------+

+-----------------------------------------------------------------+
| BUY                            | SELL                           |
| Id       | Volume      | Price | Price | Volume      | Id       |
+----------+-------------+-------+-------+-------------+----------+
|    100324|       50,000|     99|    101|       20,000|    100323|
+----------+-------------+-------+-------+-------------+----------+

+-----------------------------------------------------------------+
| BUY                            | SELL                           |
| Id       | Volume      | Price | Price | Volume      | Id       |
+----------+-------------+-------+-------+-------------+----------+
|    100324|       50,000|     99|    100|       10,000|    100325|
|          |             |       |    101|       20,000|    100323|
+----------+-------------+-------+-------+-------------+----------+

+-----------------------------------------------------------------+
| BUY                            | SELL                           |
| Id       | Volume      | Price | Price | Volume      | Id       |
+----------+-------------+-------+-------+-------------+----------+
|    100324|       50,000|     99|    100|       10,000|    100325|
|          |             |       |    101|       20,000|    100323|
|          |             |       |    100|        7,500|    100326|
+----------+-------------+-------+-------+-------------+----------+

+-----------------------------------------------------------------+
| BUY                            | SELL                           |
| Id       | Volume      | Price | Price | Volume      | Id       |
+----------+-------------+-------+-------+-------------+----------+
|    100324|       50,000|     99|    100|       10,000|    100325|
|    100327|       25,500|     98|    101|       20,000|    100323|
|          |             |       |    100|        7,500|    100326|
+----------+-------------+-------+-------+-------------+----------+

100327,100325,100,10000
100327,100326,100,7500
+-----------------------------------------------------------------+
| BUY                            | SELL                           |
| Id       | Volume      | Price | Price | Volume      | Id       |
+----------+-------------+-------+-------+-------------+----------+
|    100327|       10,000|    100|    101|       20,000|    100323|
|    100327|       25,500|     98|       |             |          |
|    100324|       50,000|     99|       |             |          |
+----------+-------------+-------+-------+-------------+----------+

